src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { removeTodo } from '../../api/todos';
+import { PossibleError } from '../../types/PossibleError';
+
+type Props = {
+  todo: Todo;
+  deleteTodo: (todoId: number) => void;
+  showError: (possibleError: PossibleError) => void;
+  hideError: () => void;
+  isLoading: boolean;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  deleteTodo,
+  showError,
+  hideError,
+  isLoading,
+}) => {
+  const [isEdited] = useState(false);
```

Comment:
do you need state here? as I can see you didn't change this value, could it be just const?